maggy/maggy/review/github.py

```
@@ -0,0 +1,169 @@
+        f"{API}{path}",
+        data=json.dumps(body).encode() if body else None,
+        method=method,
+        headers={
+            "Authorization": f"Bearer {token}",
+            "Accept": accept,
+            "X-GitHub-Api-Version": "2022-11-28",
+            "User-Agent": "revir-agentic-reviewer",
+            **({"Content-Type": "application/json"} if body else {}),
+        },
+    )
+    with urllib.request.urlopen(req) as r:
```

Ruff (0.15.17)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)